<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Admin" name="com.techpanda.admin.Admin" tests="4" failures="0" timestamp="10 Feb 2023 11:59:47 GMT" time="37.189" errors="1">
  <testcase name="TC_01_Login_With_Empty_User_Password" time="6.251" classname="com.techpanda.admin.Admin"/>
  <testcase name="TC_02_Login_With_Invalid_User_Password" time="5.084" classname="com.techpanda.admin.Admin"/>
  <testcase name="TC_04_Search_With_Emty_All_Of_Customer" time="12.752" classname="com.techpanda.admin.Admin">
    <error type="java.lang.ClassCastException" message="java.util.ArrayList cannot be cast to org.openqa.selenium.WebElement">
      <![CDATA[java.lang.ClassCastException: java.util.ArrayList cannot be cast to org.openqa.selenium.WebElement
at commons.BasePage.isListElementDisPlayedInDom(BasePage.java:336)
at pageObject.admin.AdminManageCustomerPageObject.isCustomerInforDislayed(AdminManageCustomerPageObject.java:45)
at com.techpanda.admin.Admin.TC_04_Search_With_Emty_All_Of_Customer(Admin.java:100)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC_04_Search_With_Emty_All_Of_Customer -->
  <testcase name="TC_03_Login_With_Valid_User_Password" time="13.102" classname="com.techpanda.admin.Admin"/>
</testsuite> <!-- com.techpanda.admin.Admin -->
